Adamant/Modules/Settings/Notifications/NotificationSounds/NotificationSoundsViewModel.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  NotificationSoundsViewModel.swift
+//  Adamant
+//
+//  Created by Yana Silosieva on 20.08.2024.
+//  Copyright © 2024 Adamant. All rights reserved.
+//
+
+import SwiftUI
+import CommonKit
+import Combine
+import AVFoundation
+
+@MainActor
+final class NotificationSoundsViewModel: ObservableObject {
+    private let notificationsService: NotificationsService
+    private var notificationTarget: NotificationTarget
+    private let dialogService: DialogService
+    
+    private(set) var dismissAction = PassthroughSubject<Void,Never>()
```

Comment:
Why var?

Reply:
Since dismissAction is a PassthroughSubject, which should be able to send events (and change) throughout the lifecycle of the object, using var is more appropriate. Do you think it's better to write:  let dismissAction = PassthroughSubject<Void,Never>() ?

Reply:
It's the reference type object, so of course `let`